Automattic/wp-calypso
# Stats: Deprecate feature flag "stats/new-main-chart" on the legacy HTML structure of pages

#### Proposed Changes

* Refactor shared class names to deprecate the feature flag.
* Deprecate the legacy HTML structure on the `Traffic`, `Store`, and `Ads` pages.

#### Testing Instructions



> Note: The instructions to show the Store and Ads pages can be found at https://github.com/Automattic/wp-calypso/pull/69661 and https://github.com/Automattic/wp-calypso/pull/69743.

* Spin up this change with the Calypso Live link.
* Navigate to the Stats `Traffic`, `Store`, and `Ads` pages.
* Ensure the styles are the same as the previous ones.

#### Pre-merge Checklist





Related to #70209
